### PR TITLE
Tide introduction notes

### PR DESCRIPTION
**Pull request by Ressetkk**
**Description**
Notes from the meeting about Tide introduction

### CODE REVIEW

File: docs/prow/tide-introduction-notes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+- Skipped jobs will not be reported to the GitHub. `run_if_changed` jobs will be seen as optional to GitHub, but Tide will require them before merging (hence why the tide context is set to required)
+
+### Workflow
+
+1. Developer creates PR
+2. After the required jobs have passed the Developer asks for approvals from the code owners.
+3. Once the PR has all the required approvals Tide adds PR to the merge pool.
+4. Before merge, if the dev branch is older than the latest main tide re-runs the jobs against rebased dev branch.
+5. Once the checks have passed again the PR is automatically merged.
+6. If the developer does not want the PR to be merged automatically one has to use the `/hold` command which adds the label that blocks automatic merging. To remove this label just simply use `/hold cancel`.
```
Comment (kazydek) [inline docs/prow/tide-introduction-notes.md:28]:
```suggestion
6. If you don't want the PR to be merged automatically, you have to use the `/hold` command to your PR. This command adds the label that blocks automatic branch merging. To remove this label, use the `/hold cancel` command.
```

File: docs/prow/tide-introduction-notes.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+![Tide workflow](./assets/prow-tide-workflow.png)
+
+### Next steps
+
+After upgrading Prow and Tide to the most up-to-date branch we would like to take further steps in increasing development qorkflow quality.
+
+1. Introduce organisational-wide label sync - static set of labels usable by automations and bot command on GitHub across the organisation.
```
Comment (kazydek) [inline docs/prow/tide-introduction-notes.md:36]:
```suggestion
1. Introduce organization-wide label sync - a static set of labels used by automation tools and the bot command on GitHub across the whole organization.
```

File: docs/prow/tide-introduction-notes.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+2. Introduce `approve` plugin - with this we'll be able to revoke all the write access from the repositories. Not doable until the `approve` plugin is rewritten.
```
Comment (kazydek) [inline docs/prow/tide-introduction-notes.md:37]:
```suggestion
2. Introduce the `approve` plugin. With it, we'll be able to revoke write access from the repositories. It's not doable until the `approve` plugin is rewritten.
```

**Review by kazydek — APPROVED** [review]
🎉

**Review by klaudiagrz — APPROVED** [review]

**Review by Halamix2 — APPROVED** [review]

**Pull request merged**